.eslintrc.json

```diff
@@ -1,23 +1,21 @@
 {
-    "env": {
-      "browser": true,
-      "es6": true,
-      "jest": true
-    },
-    "parser": "babel-eslint",
-    "parserOptions": {
-      "ecmaVersion": 2018,
-      "sourceType": "module"
-    },
-    "extends": ["airbnb-base"],
-    "rules": {
-      "no-shadow": "off",
-      "no-param-reassign": "off",
-      "eol-last": "off",
-      "arrow-parens": "off"
-    },
-    "ignorePatterns": [
-      "dist/",
-      "build/"
-    ]
-  }
\ No newline at end of file
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off",
+    "linebreak-style": ["error", "unix"]
+  },
+  "ignorePatterns": ["dist/", "build/"]
```

Comment:
Opps, we have some little changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/javascript)

![image](https://user-images.githubusercontent.com/31897434/73417568-ca44bd00-4318-11ea-8a5e-fedabd629b8c.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

I will advise you to follow what @kubilaycaglayan said, I quote;

`
What could be the reason? If you are using VSCode, you can delete this line and hover over the error(configure eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇
`
If you're still stuck on it, kindly reach out to me or @kubilaycaglayan.

Reply:
I start deploying the app in windows so that is why i get that error if you have the answer of to fix it tell me!!!
 @kubilaycaglayan you can delete this line and hover over the error(configure eslint checks locally).
at least read what the other TSE said beafore

Reply:
@Alexoid1, I can see your hard work, let's try to reproduce the error, then I can look into it.
You can also share a zoom link if you can.

Reply:
It's done

Reply:
Awesome.